src/angular/components/send/send.component.ts

```diff
@@ -10,6 +10,7 @@ import { SendView } from '../../../models/view/sendView';
 import { EnvironmentService } from '../../../abstractions/environment.service';
 import { I18nService } from '../../../abstractions/i18n.service';
 import { PlatformUtilsService } from '../../../abstractions/platformUtils.service';
+import { SearchService } from '../../../abstractions';
```

Comment:
Need to use the full path here to the `search.service` module to follow convention.

Reply:
Fixed, along with the other linter error